app/code/Transoft/Blog/view/frontend/templates/blogListTmpl.phtml

```diff
@@ -0,0 +1,24 @@
+<?php
+use Transoft\Blog\ViewModel\Bloglist;
+
+/** @var \Transoft\Blog\Block\Bloglist $block*/
+/** @var Bloglist $bloglistViewModel */
+
+$bloglistViewModel = $block->getData('bloglist_view_model');
+$data = $bloglistViewModel->getPostCollection()->getData();
+$match = $bloglistViewModel->isTypesMatching();
+
+?>
+<?php if ($match): ?>
+    <div>
+        <h3><?= /* @noEscape */ __('Latest 5 blogs:') ?></h3>
```

Comment:
Why  @noEscape? Please refer to [XSS prevention strategies](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/xss-protection.html)

Reply:
fixed